homework/index.js

```diff
@@ -30,13 +30,118 @@
     return elem;
   }
 
+  function setInnerText(repository, nthChild, text) {
+    document.querySelector('tbody').childNodes[nthChild].lastChild.innerText = text;
+  }
+
+  function presentRepositories(leftHandColumn, repository) {
+    const table = createAndAppend('table', leftHandColumn);
+    const tBody = createAndAppend('tbody', table);
+    const headsOfDetails = ['Repository:', 'Description:', 'Forks:', 'Updated:'];
+    for (let i = 0; i < 4; i++) {
+      const tRow = createAndAppend('tr', tBody);
+      for (let j = 0; j < 2; j++) {
+        createAndAppend('td', tRow);
+        tRow.childNodes[0].innerText = headsOfDetails[i];
+      }
+    }
+    const tData = document.querySelector('tr').lastChild;
+    createAndAppend('a', tData, {
+      href: repository.html_url,
+      target: '_blank',
+      id: 'link-repo',
+    });
+    document.getElementById('link-repo').textContent = repository.name;
+
+    const date = `${new Date(repository.updated_at).toLocaleDateString()}, ${new Date(
+      repository.updated_at,
+    ).toLocaleTimeString()}`;
+
+    setInnerText(repository, 1, repository.description);
+    setInnerText(repository, 2, repository.forks);
+    setInnerText(repository, 3, date);
+  }
+
+  function renderContributors(rightHandColumn, contributors) {
+    contributors.forEach(contributor => {
+      const ul = createAndAppend('ul', rightHandColumn);
+      const li = createAndAppend('li', ul);
+      createAndAppend('img', li, { src: contributor.avatar_url, class: 'image', height: '48' });
+      const contributorData = createAndAppend('div', li, { class: 'contributor-data' });
+      createAndAppend('div', contributorData, {
+        text: contributor.login,
+        href: contributor.html_url,
```

Comment:
Do you need to have this `href` on a `div` element?

Reply:
`li.addEventListener('click', () => {
        window.open(contributor.html_url, '_blank');
      });`

I wrote this code after writing href. I forgot to delete it. I deleted it now. Thanks Yash. :pray: